ecs_helper/cmd/pact_tags/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"os"
+	"fmt"
+	"net/http"
+	"encoding/base64"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+)
+
+func basicAuth(username, password string) string {
+	auth := username + ":" + password
+	return base64.StdEncoding.EncodeToString([]byte(auth))
+}
+
+func getPactPassword(roletoassume string, pactsecretkey string) string {
+	mysession := session.Must(session.NewSession())
+	creds := stscreds.NewCredentials(mysession, roletoassume)
+	cfg := aws.Config{Credentials: creds,Region: aws.String("eu-west-1")}
+	sess := session.Must(session.NewSession(&cfg))
+	sm := secretsmanager.New(sess)
+	secretkey := pactsecretkey
+	pact_password, err := sm.GetSecretValue(&secretsmanager.GetSecretValueInput{SecretId: &secretkey})
+	if err != nil {
+		panic(err.Error())
+	}
+	return *pact_password.SecretString
+}
+
+func main() {
+
+	base_url := os.Getenv("PACT_BROKER_BASE_URL")
```

Comment:
As above, it's best to stick to one type of naming convention for env vars to aid readability.